tartufo.toml

```diff
@@ -4,18 +4,21 @@ cleanup = true
 regex = true
 entropy = true
 
-exclude-path-patterns = [
-  'tartufo\.toml$',
-  'package-lock\.json$',
-  'CHANGELOG\.md$'
-]
+  [[tool.tartufo.exclude-path-patterns]]
+  path-pattern = "tartufo\\.toml$"
```

Comment:
this change was required, was getting an error `tartufo.types.ConfigException: String pattern is illegal in exclude-path-patterns`

Reply:
I have #17 and #18 to hopefully address any needs here.